qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -468,6 +468,11 @@ def usage_estimation(self) -> Dict[str, Any]:
 
         return self._usage_estimation
 
+    @property
+    def instance(self) -> Optional[str]:
+        """Return the instance the job was run on"""
```

Comment:
"Instance" is an IQP term, not for the pay-go cloud channel. What happens if this is a cloud channel job? Does this blow up? Does it return null or an empty string?

Reply:
Returns `None` for cloud channel jobs - I'll update the docstring to be more clear